connectivity/access_jobs.sql

```diff
@@ -3,40 +3,38 @@
 -- location: cambridge
 ----------------------------------------
 -- low stress access
-UPDATE  cambridge_census_blocks
+UPDATE  neighborhood_census_blocks
 SET     emp_low_stress = (
             SELECT  SUM(blocks2.jobs)
             FROM    cambridge_census_block_jobs blocks2
```

Comment:
Do this line and others like it need to be updated? Should references to cambridge be removed?